scripts/create-clusters.sh

```diff
@@ -114,22 +113,22 @@ function create-clusters() {
 function fixup-cluster() {
   local i=${1} # cluster num
 
-  local kubeconfig_path="$(kind get kubeconfig-path --name ${i})"
+  local kubeconfig_path="$(kind get kubeconfig-path --name cluster${i})"
   export KUBECONFIG="${KUBECONFIG:-}:${kubeconfig_path}"
 
   # Simplify context name
-  kubectl config rename-context kubernetes-admin@kind-${i} cluster${i}
+  kubectl config rename-context "kubernetes-admin@cluster${i}" "cluster${i}"
 
   # TODO(font): Need to set container IP address in order for clusters to reach
   # kube API servers in other clusters until
   # https://github.com/kubernetes-sigs/kind/issues/111 is resolved.
-  local container_ip_addr=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kind-${i}-control-plane)
-  sed -i "s/localhost/${container_ip_addr}/" ${kubeconfig_path}
+  local container_ip_addr=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster${i}-control-plane)
+  sed -i "s/localhost.*$/${container_ip_addr}:6443/" ${kubeconfig_path}
```

Comment:
why?

Reply:
I've added a comment documenting the change.  PTAL.